pkg/utils/strings.go

```diff
@@ -33,3 +37,8 @@ func convertSliceToInterface(s interface{}) (slice []interface{}) {
 
 	return slice
 }
+
+// StringJoin takes two strings and joins them with defaultConnector
+func StringJoin(prefix string, suffix string) string {
+	return prefix + defaultConnector + suffix
```

Comment:
Why not https://golang.org/pkg/strings/#Join, e.g. `strings.Join(botName, wip.ProwPluginName, '/')` ?